raiden/raiden_event_handler.py

```diff
@@ -90,6 +232,31 @@ def unlock(
         log.error(str(e), node=pex(raiden.address))
 
 
+def unlock_light(raiden: "RaidenService",
+                 chain_state,
+                 channel_unlock_event,
+                 participant: Address,
+                 partner: Address):
+    canonical_identifier: CanonicalIdentifier = channel_unlock_event.canonical_identifier
+
+    token_network: TokenNetwork = views.get_token_network_by_identifier(chain_state, canonical_identifier.token_network_address)
+
+    LightClientMessageHandler.store_light_client_protocol_message(
```

Comment:
Here we should check for duplicated messages before adding it to the database. I have created a new function to check this on the branch ```feature/RLP-748/settlement_light``` maybe you can use it to check for duplicated messages and if the message it's already on database you should skip this. 

When you can have duplicated messages?
When you have a lumino node that is down for some reason and when you start it again it recreates all the state events again. Then you have this code running again and adding a new message with the same data.